mlflow/store/artifact/runs_artifact_repo.py

```diff
@@ -156,6 +157,16 @@ def download_artifacts(self, artifact_path, dst_path=None):
                     # Return the first model that matches the run_id and artifact_path
                     if model.source_run_id == run_id and model.name == artifact_path:
                         repo = get_artifact_repository(model.artifact_location)
+                        color_warning(
+                            "`runs:/<run_id>/artifact_path` is deprecated for loading models, "
+                            "use `models:/<model_id>` instead. Alternatively, retrieve "
+                            "`model_info.model_uri` from the model_info returned by "
+                            "mlflow.<flavor>.log_model. For example:\n"
```

Comment:
```suggestion
                            "mlflow.<flavor>.log_model. For example: "
```

Can we remove `\n` here?